zwave_js_server/model/firmware.py

```diff
@@ -13,6 +13,67 @@
     from .node import Node
 
 
+class FirmwareUpdateCapabilitiesDataType(TypedDict, total=False):
+    """Represent a firmware update capabilities dict type."""
+
+    firmwareUpgradable: bool  # required
+    firmwareTargets: List[int]
+    continuesToFunction: Union[bool, str]
+    supportsActivation: Union[bool, str]
+
+
+class FirmwareUpdateCapabilities:
+    """Model for firmware update capabilities."""
+
+    def __init__(self, node: "Node", data: FirmwareUpdateCapabilitiesDataType) -> None:
+        """Initialize class."""
+        self.node = node
```

Comment:
Why do we set the node here? It's not used.

Reply:
I thought it was needed when I first started but you are right, I'll remove